utils/pull_requests.py

```diff
@@ -10,12 +10,18 @@
 token_header = dict()
 keyword = None
 
-
 def eprint(*args, **kwargs):
     print(*args, file=sys.stderr, **kwargs)
 
+def author_in_whitelist(author, options, pr_no):
```

Comment:
I would pass options.whitelist to this method instead of options since no other arg is used from options and it makes it clearer what the method is acting on.

Reply:
TBH, that's how I had it before.  Then I started just passing ``options`` through to other functions, so I did it here for consistency-sake.  Though you're right, it's not strictly needed.  I'll remove it if that's a hard-requirement, otherwise I'm more comfortable with being lazy 😀

Reply:
it makes sense in the other areas where you are using multiple options.  Put it this way, if you were to put this in a library would you expect your callers to wrap that argument in an options object just so they can call your method?